AcademyChallenge/DataBase/ReposCoreData.swift

```diff
@@ -0,0 +1,89 @@
+//
+//  ReposEntity.swift
+//  AcademyChallenge
+//
+//  Created by Keven Esmael on 17/10/2022.
+//
+
+import Foundation
+import UIKit
+import CoreData
+
+class ReposCoreData {
+	
+	var ReposPersistence: [NSManagedObject] = []
+   
+   var appDelegate: AppDelegate
```

Comment:
Keeping a reference of the AppDelegate isn't necessary and can lead to some retain cycle issues

Reply:
solved